src/uiTest/resources/ideprobe.conf

```
@@ -1,5 +1,6 @@
 probe {
   paths.screenshots = ${?IDEPROBE_SCREENSHOTS_DIR}
```

Nit: I'd actually keep the env var names as aligned with the corresponding config keys as possible:
```suggestion
  paths.screenshots = ${?IDEPROBE_PATHS_SCREENSHOTS}
```